apps/e2e-test/features/publisher-loacl-file-preview.feature

```diff
@@ -139,7 +139,7 @@ Feature: Publisher Local File Preview
 
         When the publisher hovers the mouse over the "local file view video button"
         Then the "tooltip" should be displayed
-        And the "tooltip" text should be "Toggle camera"
+        And the "tooltip" text should be "Toggle video"
```

Comment:
please rename filename typo from `loacl` to `local`

Reply:
@jackshen Fixed